style2.css

```diff
@@ -0,0 +1,545 @@
+@media only screen and (min-width: 768px) {
+  .ham-manu {
+    display: none;
+  }
+
+  .nav-bar {
+    height: 4.5em;
+  }
+
+  .nav-menu {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .nav-menu-item {
+    margin: 0 2.62em 0 0;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-size: 0.93rem;
+    font-weight: 600;
+  }
+
+  .logo {
+    margin: 0 0 0 2.5em;
+  }
+
+  .logo-txt {
+    font-family: 'Inter', sans-serif;
+    font-size: 1.25rem;
+  }
+
+  .hero-section {
+    background-image: url(res/desktop-Illustration-header.svg);
+    background-position: right top 50%;
+    background-size: 50vw;
+  }
+
+  .hero-main-header {
+    padding: 13em 0 0 5em;
+    margin-right: 1em;
+  }
+
+  .hero-main-header h1 {
+    font-size: 3.5rem;
+  }
+
+  .hero-sm-header {
+    padding: 0.75em 0 0 5em;
+    margin-right: 1em;
+  }
+
+  .hero-sm-header h3 {
+    font-weight: 500;
+  }
+
+  .hero-text {
+    padding: 0.75em 0 0 5em;
+    width: 30em;
+    margin-right: 1em;
+  }
+
+  .hero-text p {
+    font-weight: regular;
+  }
+
+  .social-media-sec {
+    display: inline-block;
+    position: relative;
+    bottom: 14em;
+  }
+
+  .social-ico {
+    padding-bottom: 1em;
+  }
+
+  .works-section {
+    display: grid;
+    width: 90vw;
+    margin: 0 auto;
+  }
+
+  .my-recent-work {
+    margin: 8rem auto 0 auto;
+    display: flex;
+  }
+
+  .my-recent-work h1 {
+    text-align: center;
+    margin: 0 1rem 0 0;
+  }
+
+  .recent-hr {
+    width: 40vw;
+    margin: auto -2em 6em 0;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 0 0.5em;
+    grid-template-areas:
+      "project-1 project-1 project-1"
+      "project-2 project-3 project-4"
+      "project-5 project-6 project-7";
+  }
+
+  .project-1 {
+    grid-area: project-1;
+    display: grid;
+    grid-template-areas:
+      "project-1-img project-1-txt"
+      "project-1-img project-1-txt";
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto 0 1em;
+  }
+
```

Comment:
- [OPTIONAL ] I noticed that you made some modifications, but you still need to improve your design. Because we are emulating the Figma design, and your design and the Figma design are not really comparable, please adjust your design to look like the Figma one so that your project would seem fantastic. :wink: 

- [ ] I also suggest you increase the width of your card image on the desktop version to match the Figma design, kindly fix that also 

Your design :point_down: 
![error](https://user-images.githubusercontent.com/88894525/194531426-993f5d67-7446-4c71-8a7b-ab7aba7032dc.png)

Figma
![followthiscard](https://user-images.githubusercontent.com/88894525/194531507-492fb3f7-3888-48b4-bd28-5e83b7042fbb.png)